client/ayon_core/plugins/publish/extract_hierarchy_to_ayon.py

```diff
@@ -22,7 +22,7 @@ class ExtractHierarchyToAYON(pyblish.api.ContextPlugin):
 
     order = pyblish.api.ExtractorOrder - 0.01
     label = "Extract Hierarchy To AYON"
-    families = ["clip", "shot"]
+    families = ["clip", "shot", "csv_ingest_shot"]
```

Comment:
The same as in Collect Hierarchy, it is context plugin, does not need families filtering.
```suggestion
```